lib/inspec/env_printer.rb

```diff
@@ -35,7 +35,7 @@ def print_and_exit!
     private
 
     def print_completion_for_shell
-      erb = ERB.new(File.read(completion_template_path), nil, "-")
+      erb = ERB.new(File.read(completion_template_path), trim_mode: "-")
```

Comment:
Some explanation about the change. The old way of calling used positional arguments to indicate using a default safe level (nil), then went on to indicate the trim mode "-" (which is one of the modes you can select).

The new way of saying the same thing using named arguments would be to omit the nil and to say `trim_mode: "-"`.